tex/texmaker/Portfile

```
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           qmake5 1.0
+
+name                texmaker
+version             5.0.4
+revision            0
+categories          tex editors
+platforms           darwin
+license             GPL-2
+
+maintainers         {@harens gmail.com:harensdeveloper} \
+                    openmaintainer
+
+description         Free cross-platform LaTeX editor since 2003
+long_description    Texmaker is a free, modern and cross-platform LaTeX editor \
+                    that integrates many tools needed to develop documents with LaTeX, \
+                    in just one application.
+
+master_sites        https://www.xm1math.net/texmaker/
+homepage            ${master_sites}
+
+distfiles           ${name}-${version}.tar.bz2
```

This will only change distfiles, not the extract command. That will still work on OS X 10.9 and later but not on 10.8 or earlier. Instead, use `use_bzip2 yes` which will change both.
```suggestion
use_bzip2           yes
```

---

Thanks @ryandesign. I've added your changes and noted you as a co-author.